shell/platform/fuchsia/flutter/fuchsia_external_view_embedder.h

```diff
@@ -131,6 +132,37 @@ class FuchsiaExternalViewEmbedder final : public flutter::ExternalViewEmbedder {
     scenic::Material material;
   };
 
+  // Helper class for setting up an invisible rectangle to catch all input. Rejected input will then
+  // be re-injected into the scene below us.
```

Comment:
" into the scene below us." -> "into a suitable platform view controlled by this Engine instance."